app/redux/ducks/classify.spec.js

```diff
@@ -459,22 +459,26 @@ describe('Classifier actions', function () {
         }
       }
     };
-    const state = {
-      classification: mockPanoptesResource('classifications', {
-        id: '1',
-        metadata: {
-          b: 3,
-          c: 4
-        }
-      }),
-      workflow: {
-        id: '1',
-        tasks: {
-          a: {}
-        }
-      },
-      upcomingSubjects: [mockSubject('1'), mockSubject('2')]
-    };
+    // ensure state is reset after each test
+    let state = null;
```

Comment:
If you want to ensure this is happening after each test, then I think the safest way to do this is to use `afterEach(function () { state = null }`  within the describe block too.

Reply:
This was working as expected, the comment was the problem as it was slightly misleading. 

`let state = null;` sets up the state variable for the describe block, the state itself is now reset in the `beforeEach` function on line 465 instead of having state carry over from one test to the other. 

I.e. the test setup is now clean whereas before state modifications through redux actions polluted the next test.

Reply:
In that case, can you declare the state at the top of the block, so it's clear that it's set at the same time as the test functions are declared.

Reply:
Can do - personally i think this is an issue with how the GH UI displays the change set as when you look at the code it's pretty obvious it's a variable setup declaration.
<img width="630" alt="Screenshot 2019-06-12 at 10 25 21" src="https://user-images.githubusercontent.com/295329/59339751-a89cb400-8cfc-11e9-8348-2b2e4bfa4387.png">

Reply:
Yeah, I misread it as being after the `it()` functions. My bad.

Reply:
Oh same. I've been bitten by how github displays diffs in the past too. Gotta remember to view it as the full file.